docs/settings/tts/watson.md

```diff
@@ -16,9 +16,11 @@ Once you project created, you should see your credentials like the following
 
 | Parameters | Required | Default | Choices               | Comment                                           |
 |------------|----------|---------|-----------------------|---------------------------------------------------|
-| apikey     | yes      |         |                       | apikey provided by [IAM](https://console.bluemix.net/docs/services/watson/getting-started-iam.html)                 |
-| location   | no       |  LONDON |                       | [endpoint location](https://www.ibm.com/watson/developercloud/text-to-speech/api/v1/curl.html?curl#service-endpoint)                               |
```

Comment:
Can you please keep the formatting of markdown table? I recommend you to use a plugin like markdown all in one if you are a vscode user.
![image](https://user-images.githubusercontent.com/2573405/99256125-34c9d980-2815-11eb-8039-60aa1a737ddf.png)